README.md

```diff
@@ -13,7 +13,8 @@ reward-driven, platform for goods and service exchanges
 
 ### Supply
 
-Differently from traditional currencies, such as Bitcoin, Litecoin and Ethereum,
+<!-- -1: Ethereum is not deflationary -->
+Differently from traditional currencies, such as Bitcoin, Litecoin and Ethereum[-1],
```

Comment:
Ethereum is still minting more coins than they are burning, even after EIP-1559

https://www.theblockcrypto.com/post/113650/heres-how-much-eth-has-already-been-burned-due-to-eip-1559